Add an initial source repository fetcher for openSUSE repos

boiko opened the pull request:
This still needs improvements, but could already be used to generate a test repo for the development to continue

boiko committed:
Add an initial source repository fetcher for openSUSE repos

This still needs improvements, but could already be used to generate a test repo
for the development to continue

oSoMoN commented on source_pool_fetcher.py:
```diff
@@ -0,0 +1,137 @@
+        """
+        Fetch new versions of sources to store locally.
+        """
+        pass
```
You might want to `raise NotImplementedError()` here?

oSoMoN reviewed (approved):
LGTM! That's concise and elegant. Just one minor suggestion inline, and also maybe we could add some logging to make it easier to debug?
But feel free to merge as is and update later on.

boiko committed:
Raise NotImplementedError on the abstract methods of the base class

boiko committed:
Use the non-crashing version of rpmfile for the time being

Hopefully the upstream change will be accepted/reworked:
https://github.com/srossross/rpmfile/pull/45

boiko committed:
Add more logging

And use debug as the default at least until we are done with the initial development
